operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -49,14 +70,25 @@ local utils = import 'mixin-utils/utils.libsonnet';
             },
           ],
           [
+            'memory',
```

Opinionated ignorable nit:

```suggestion
            'in-memory',
```

`memory` is understood, but it's not consistent with the rest of references to this.

---

Works for me